libr/core/cmd_meta.c

```diff
@@ -414,8 +414,20 @@ static int cmd_meta_hsdmf(RCore *core, const char *input) {
 
 	switch (input[1]) {
 	case '?':
-		eprintf ("See C?\n");
-		break;
+		switch (input[0]) {
+		case 'f':
+			r_cons_printf ("Usage: Cf[-] [sz] [fmt..] [@addr]\n\n");
```

Comment:
you can do this with just one r_cons_printf.. and in fact you dont need printf at all here, because this is not formatting anything, use r_cons_println() and just do "foo" \n "bar" \n ...